packages/tea-map-cache/package.json

```diff
@@ -7,7 +7,8 @@
     "node": ">=12.18.0"
   },
   "scripts": {
-    "package": "rm -Rf ./dist/ && mkdir dist && zip -r -q ./dist/lambda.zip *.js node_modules",
+    "build": "tsc",
```

Comment:
Don't we need a `tsc` script here that calls out to the `tsc` package in the top level directory? I'm just basing this on how we've done the other packages. Also a `clean` script

Reply:
Right you are: ebcbac7a1.    I'm assuming I just hadn't yet seen the CI failure this would cause.